cobalt/shell/android/java/src/dev/cobalt/shell/Shell.java

```
@@ -174,7 +183,7 @@ public void loadUrl(String url) {
      * @return The sanitized URL.
      */
     public static String sanitizeUrl(String url) {
-        if (url == null) return null;
+        assert url != null;
```

This gets stripped away in release builds by R8. If we remove the if check but keep the assert, we lose some of the check functionality. If we want this to throw an exception in all builds, we should throw an exception such as a `IllegalArgumentException`. If you want to keep the assert there for debug builds, you should wrap it in a block:

```
if (BuildConfig.ENABLE_ASSERTS) {
    // Any code here will be stripped in release builds by R8.
    ...
}
```

and then in the else block, throw an exception so release builds also get an exception thrown during runtime. For simplicity, I would recommend just throwing an exception if we expect this to always be non-null.

Source - https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/java/java.md#asserts